### PR TITLE
Feature/rectangle roi mobile 1.7.3

### PR DESCRIPTION
**Pull request by syzer**

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "dicom-parser": "file:../node_modules/dicom-parser",
     "hammerjs": "file:../node_modules/hammerjs",
     "react": "file:../node_modules/react",
-    "react-cornerstone-viewport-hooks": "file:..",
+    "react-cornerstone-viewport-hooks": "file:../dist/index.modern.js",
```
Comment (syzer) [inline example/package.json:20]: here was to get reloading behaviour of webpack...

File: example/src/PageRectangleRoiUniqueDemo.js [hunk 2]
```diff
@@ -46,6 +44,7 @@ const PageRectangleRoiUniqueDemo = function () {
       <h2>RectangleRoiUniqueTool Demo</h2>
       <div style={{ display: "flex", flexWrap: "wrap" }}>
         <ReactCornerstoneViewportHooks
+          activeToolName={ReactCornerstoneViewportHooksHelpers.TOOL_NAMES.RectangleRoiUnique}
```
Comment (syzer) [inline example/src/PageRectangleRoiUniqueDemo.js:47]: this is required, because default is to `Wwwc`

File: example/src/util.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { ReactCornerstoneViewportHooksHelpers } from "react-cornerstone-viewport-hooks";
+
+export const useDebugToolState = () => useEffect(() => {
```
Comment (syzer) [inline example/src/util.js:4]: print toolstate to console quite important for our backend,
added beecause sometimes was not stored correctly

File: src/ReactCornerstoneViewportHooksHelpers.js [hunk 4]
```diff
@@ -94,7 +94,7 @@ export const setActiveTool = function (
 
   if (!validToolNames.includes(activeToolName)) {
     console.warn(
-      `Trying to set a tool active that is not "added". Available tools include: ${validToolNames.join(
+      `Trying to set a tool active '${activeToolName}' that is not "added". Available tools include: ${validToolNames.join(
```
Comment (syzer) [inline src/ReactCornerstoneViewportHooksHelpers.js:97]: here fixed to print out hte tool it's trying to add.. figured out it was wwwc

File: example/package.json (hunk 1 above)
Comment (wheresvic) [inline example/package.json:20]: Why this change?

**Comment by wheresvic** [comment]
@syzer there are ~30k deletions in package-lock - where are they coming from?

File: src/RectangleRoiMobileTool.js [hunk 5]
```diff
@@ -70,13 +196,19 @@ export default class RectangleRoiMobileTool extends BaseTool {
     const element = evt.detail.element;
     const image = evt.detail.currentPoints.image;
 
+    // console.warn('_startOutliningRegion', this.isAnnotating)
```
Comment (syzer) [inline src/RectangleRoiMobileTool.js:199]: I leave it for now... as commented out code
it's important for refresh of drawing box

File: example/package.json (hunk 1 above)
Comment (wheresvic) [reply example/package.json:20]: ok got it

File: src/RectangleRoiMobileTool.js (hunk 5 above)
Comment (wheresvic) [reply src/RectangleRoiMobileTool.js:199]: maybe put in a comment in the code?

**Comment by syzer** [comment]
@wheresvic don't know.. npm resolution.. I guess only meaningful change I see is 1.7.2 => 1.7.3


If you are not sure you can always `npm i` yourself and see the diff
